addons/mail/static/src/js/systray/systray_activity_menu.js

```diff
@@ -127,6 +127,20 @@ var ActivityMenu = Widget.extend({
         }
     },
 
+    /**
+     * Return views to display when coming from systray depending on the model.
+     * @private
+     */
+    getViewsList: function(model){
```

Comment:
- method should be private if not called from outside
- it should be placed alphabetically with other methods, while respecting the public/private/handlers sections
- docstring should also have the `@private` then
- it's also missing the appropriate `@param` and `@returns`
- use shorthand syntax:
```suggestion
    _getViewsList(model) {
```